wiris/html-integrations
# fix: insert buttons are hidden on ipad

## Description

 In this Issue the insert buttons was outside of the user window, having scroll, and not working consistently. This "kindoff" redesigns the styles of the modal and changes some of the old behavior to fit all screens. This includes Android and apple devices. 

- **Related Kanbanize Card:** [Link to KB-51855](https://wiris.kanbanize.com/ctrl_board/2/cards/51855/details/)

## Type of Change


## Checklist


## How should be tested? (Manual or Automated Tests)

On this [url](https://integrations.wiris.kitchen/fix/KB-51855/ipad-fullscreen/), check using your phone / tablet to see if everything fits on screen
